### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by rachitjain2706**

### CODE REVIEW

**Review by sahildeshpande25 — APPROVED** [review]
Looks good! Clear descriptions provided about the UI.

File: README.md [hunk 1]
```diff
@@ -60,6 +60,56 @@ tests/              - test cases package
...
+<p align=center>
+<img src="images/ui.png" alt="isolated" />
+</p>
+<h4 align="center">
+
+#### `Create an Inventory Item`
+- Set the required fields of the inventory item.
+- Press `Create` and you should get a success message with the `Item ID` field populated.
+
+### `Retrive an Inventory Item`
```
Comment (sahildeshpande25) [inline README.md:82]:
```suggestion
### `Retrieve an Inventory Item`
```

File: README.md [hunk 2]
```diff
@@ -60,6 +60,56 @@ tests/              - test cases package
...
+## User Interface
+
+Following is the User Interface that is provided to the inventory manager when the home page opens.
+<h4 align="center">
+<p align=center>
+<img src="images/ui.png" alt="isolated" />
+</p>
+<h4 align="center">
+
+#### `Create an Inventory Item`
```
Comment (zhengtr) [inline README.md:78]:
```suggestion
### `Create an Inventory Item`
```

File: images/ui.png
Comment (zhengtr) [inline images/ui.png:1]: Subject to be changed after PR #107 is merged.

**Comment by zhengtr** [comment]
Would you please also update the contents (tree structure) of our project? Maybe add `/service/static` and `/features`.

**Review by zhengtr — APPROVED** [review]
LGTM! Nicely done.

**Comment by rachitjain2706** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/15332261-faa1-4111-91e5-bb4a4e7761cb?env_id=ibm:yp:us-south "nyu-inventory"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/0ee75414-6b60-4df5-9cdb-fdf2afb5159e "inventory") deployed [inventory](https://github.com/CSCI-GA-2820-SP23-003/inventory/releases/tag/deploy-inventory-20230501-050813 "deploy-inventory-20230501-050813") to [inventory](https://cloud.ibm.com/kubernetes/clusters/ch6nt2if025pha8qe010/overview "There was a problem retrieving the space name"), including commit 7ace092f52e88156c5d21b365bdc3ed4c6f47765

**Pull request merged**